src/store/RelayContext.js

```diff
@@ -0,0 +1,269 @@
+/**
+ * Copyright 2013-2015, Facebook, Inc.
+ * All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree. An additional grant
+ * of patent rights can be found in the PATENTS file in the same directory.
+ *
+ * @providesModule RelayContext
+ * @typechecks
+ * @flow
+ */
+
+'use strict';
+
+const GraphQLFragmentPointer = require('GraphQLFragmentPointer');
+import type RelayMutation from 'RelayMutation';
+import type RelayMutationTransaction from 'RelayMutationTransaction';
+import type RelayQuery from 'RelayQuery';
+const RelayQueryResultObservable = require('RelayQueryResultObservable');
+const RelayStoreData = require('RelayStoreData');
+const RelayTaskScheduler = require('RelayTaskScheduler');
+
+const forEachRootCallArg = require('forEachRootCallArg');
+const invariant = require('invariant');
+const readRelayQueryData = require('readRelayQueryData');
+const warning = require('warning');
+
+import type {
+  Abortable,
+  Observable,
+  RelayMutationTransactionCommitCallbacks,
+  ReadyStateChangeCallback,
+  StoreReaderData,
+  StoreReaderOptions,
+} from 'RelayTypes';
+
+import type {
+  DataID,
+  RelayQuerySet,
+} from 'RelayInternalTypes';
+
+let defaultInstance;
```

Comment:
let's use `RelayStore` as the default instance, so we don't need a default instance or `getDefaultInstance` here